change/@fluentui-react-tags-preview-908ff2e5-8a31-4853-b73f-03b2320f85fe.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28712&commitId=ef99bdc14bfcf999ab4dcb78daa887e74fd0cc3f&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 94 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand Disabled - High Contrast.default.chromium.png|611| Changed|
|InteractionTag Converged.Circular With Media.default.chromium.png|397| Changed|
|InteractionTag Converged.Brand - High Contrast.default.chromium.png|829| Changed|
|InteractionTag Converged.Brand Disabled.default.chromium.png|474| Changed|
|InteractionTag Converged.Brand Disabled - Dark Mode.default.chromium.png|579| Changed|
|InteractionTag Converged.Circular With Secondary Text.default.chromium.png|393| Changed|
|InteractionTag Converged.Circular Dismissible.default.chromium.png|305| Changed|
|InteractionTag Converged.Brand.default.chromium.png|538| Changed|
|InteractionTag Converged.Circular Dismissible - RTL.default.chromium.png|313| Changed|
|InteractionTag Converged.Default - RTL.default.chromium.png|285| Changed|
|InteractionTag Converged.Default - Dark Mode.default.chromium.png|487| Changed|
|InteractionTag Converged.Default - High Contrast.default.chromium.png|600| Changed|
|InteractionTag Converged.Circular With Media - RTL.default.chromium.png|388| Changed|
|InteractionTag Converged.Default.default.chromium.png|277| Changed|
|InteractionTag Converged.Brand - Dark Mode.default.chromium.png|456| Changed|
|InteractionTag Converged.Filled - High Contrast.default.chromium.png|829| Changed|
|InteractionTag Converged.Disabled - Dark Mode.default.chromium.png|579| Changed|
|InteractionTag Converged.Disabled.default.chromium.png|474| Changed|
|InteractionTag Converged.Circular.default.chromium.png|277| Changed|
|InteractionTag Converged.Outline - Dark Mode.default.chromium.png|419| Changed|
|InteractionTag Converged.Disabled - High Contrast.default.chromium.png|611| Changed|
|InteractionTag Converged.Filled.default.chromium.png|350| Changed|
|InteractionTag Converged.Filled - Dark Mode.default.chromium.png|680| Changed|
|InteractionTag Converged.Outline Disabled - Dark Mode.default.chromium.png|257| Changed|
|InteractionTag Converged.Outline - High Contrast.default.chromium.png|829| Changed|
|InteractionTag Converged.Outline Disabled - High Contrast.default.chromium.png|611| Changed|
|InteractionTag Converged.Outline.default.chromium.png|417| Changed|
|InteractionTag Converged.Rounded Dismissible.default.chromium.png|305| Changed|
|InteractionTag Converged.Rounded Dismissible - RTL.default.chromium.png|313| Changed|
|InteractionTag Converged.Rounded With Media.default.chromium.png|408| Changed|
|InteractionTag Converged.Outline Disabled.default.chromium.png|286| Changed|
|InteractionTag Converged.Rounded With Media - RTL.default.chromium.png|398| Changed|
|InteractionTag Converged.Rounded.default.chromium.png|277| Changed|
|InteractionTag Converged.Rounded With Secondary Text.default.chromium.png|393| Changed|
|InteractionTag Converged.Size Small With Icon.default.chromium.png|271| Changed|
|InteractionTag Converged.Size Extra Small Dismissible.default.chromium.png|221| Changed|
|InteractionTag Converged.Size Extra Small With Media.default.chromium.png|284| Changed|
|InteractionTag Converged.Size Extra Small.default.chromium.png|208| Changed|
|InteractionTag Converged.Size Extra Small With Icon.default.chromium.png|250| Changed|
|InteractionTag Converged.Size Small With Media.default.chromium.png|306| Changed|
|InteractionTag Converged.Size Small Dismissible.default.chromium.png|248| Changed|
|InteractionTag Converged.Size Small.default.chromium.png|226| Changed|
|InteractionTag Converged.Brand - Dark Mode.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Brand - Dark Mode.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Brand - Dark Mode.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Brand - Dark Mode.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Brand - High Contrast.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Brand - High Contrast.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Brand - High Contrast.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Brand - High Contrast.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Brand.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Brand.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Brand.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Brand.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Circular Dismissible - RTL.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Circular Dismissible - RTL.focus dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Circular Dismissible.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Circular Dismissible.focus dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Circular With Media - RTL.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Circular With Media.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Circular With Secondary Text.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Circular.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Filled - Dark Mode.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Filled - Dark Mode.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Filled - Dark Mode.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Filled - Dark Mode.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Filled - High Contrast.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Filled - High Contrast.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Filled - High Contrast.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Filled - High Contrast.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Filled.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Filled.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Filled.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Filled.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Outline - Dark Mode.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Outline - Dark Mode.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Outline - Dark Mode.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Outline - Dark Mode.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Outline - High Contrast.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Outline - High Contrast.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Outline - High Contrast.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Outline - High Contrast.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Outline.hover content.chromium.png|0| Removed|
|InteractionTag Converged.Outline.hover dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Outline.pressed content.chromium.png|0| Removed|
|InteractionTag Converged.Outline.pressed dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Rounded Dismissible - RTL.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Rounded Dismissible - RTL.focus dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Rounded Dismissible.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Rounded Dismissible.focus dismiss.chromium.png|0| Removed|
|InteractionTag Converged.Rounded With Media - RTL.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Rounded With Media.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Rounded With Secondary Text.focus content.chromium.png|0| Removed|
|InteractionTag Converged.Rounded.focus content.chromium.png|0| Removed|

</details>


  <details><summary><b>TagGroup Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagGroup Converged.Dismissible With Interaction Tag.default.chromium.png|512| Changed|
|TagGroup Converged.Dismissible.default.chromium.png|337| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>